lib/product-import.coffee

```diff
@@ -168,7 +168,6 @@ class ProductImport
         @client.productProjections
         .where(predicate)
         .staged(true)
-        .perPage(100)
```

Comment:
This has been fixed [here] https://github.com/sphereio/sphere-node-sdk/releases/tag/v1.20.0

> why we need perPage() when we use all()

there is no way API will return you all elements. `.all()` is just SDK helper which fetches all results into memory and internally does pagination with multiple requests. If during this pagination no page limit is set then API will default to 20 per page. SDK's `.all()` currently defaults to 50 per page if non provided over `.perPage()` function. Considering that when you use `.all()` you will always load everything into memory it should be faster if you do less requests so per page should be rather higher than our defaults. It might be that API's max of 500 elements per page could be slow in some cases (very big products) but i would use here at least 200 as limit.